### PR TITLE
Implement ZoomIn/ZoomOut/ZoomReset with configurable step

### PR DESCRIPTION
**Pull request by MoSal**
Implements #14.

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -42,6 +44,8 @@ impl Default for Config {
             app_theme: AppTheme::System,
             font_name: "Fira Mono".to_string(),
             font_size: 14,
+            font_size_zoom_adj: 0,
```
Comment (jackpot51) [inline src/config.rs:47]: Should this default be 0?

Comment (jackpot51) [reply src/config.rs:47]: Looks like 0 is correct, based on the other code

**Comment by jackpot51** [comment]
This looks good, so I merged. I'm not sure if I want to save the zoom to config and, in doing so, sync it across instances. It seems to me like permanent adjustments ought to be done by changing the font size, while zoom can be used for temporary adjustments that are per-tab.

**Comment by leviport** [comment]
In my experience, zoom resets when closed in most terminal apps. Gnome Console is the only one I can recall off the top of my head that remembers zoom levels. So I think that sounds good as well.

**Pull request merged**